test/subgraph-tests/position.test.ts

```diff
@@ -7,7 +7,7 @@ describe('Order fetching logic from subgraph', () => {
     console.log('Test running successfully');
     const positionId = '0x5c46fe7154af223da5e2e6d284e367d4ef38bdfd5c6fd4ce56cc47d0d3cbd957';
 
-    const position = await getPositionById(chain, positionId);
+    const position = await getPositionById(chain, positionId, undefined);
```

Comment:
why are you sending a extra params? how is it working this [function](https://github.com/Parifi/parifi-sdk/blob/d624a9519b7846a1e690ef6aca3f0bb2efc9bb97/src/subgraph/positions/index.ts#L31) only recieve 2 params

Reply:
done